### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MaybeTI**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        file_json = json.load(f)
+        for key, value in file_json.items():
```
Comment (WDemidenko) [inline main.py:10]: Reread checklist, you dont need to write the who function body inside your context manager with open. Also use more descriptive names than key and value.

**Review by WDemidenko — CHANGES_REQUESTED** [review]

**Review by WDemidenko — APPROVED** [review]
Good job!

File: db/models.py [hunk 2]
```diff
@@ -1 +1,26 @@
...
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.SET_NULL, null=True)
```
Comment (YuriiKindrat) [inline db/models.py:24]: are u sure that player could have null as race ? 
image

File: main.py [hunk 3]
```diff
@@ -1,10 +1,55 @@
...
+            ).exists()
+            guild = (
+                Guild.objects.get(
+                    name=player_info["guild"]["name"])
+            ) if guild_exists else Guild.objects.create(
+                name=player_info["guild"]["name"],
+                description=player_info["guild"]["description"]
+            )
+        else:
+            guild = None
```
Comment (YuriiKindrat) [inline main.py:44]: no need to use else here, u can create this variable before if condition

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

File: db/models.py [hunk 4]
```diff
@@ -1 +1,26 @@
...
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE, null=True)
+    guild = models.ForeignKey(Guild, on_delete=models.SET_NULL, null=True)
+    created_at = models.DateField(auto_now=True)
```
Comment (YuriiKindrat) [inline db/models.py:26]: are u sure about auto_now here? check ur checklist

File: db/models.py [hunk 5]
```diff
@@ -1 +1,26 @@
...
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.SET_NULL, null=True)
```
Comment (YuriiKindrat) [inline db/models.py:12]: image
are u sure that skill can exist without race ?

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

**Review by YuriiKindrat — APPROVED** [review]

**Pull request closed without merging**